example/lib/main.dart

```diff
@@ -103,7 +114,7 @@ class MyApp extends StatelessWidget {
                 listTextStyle: TextStyle(fontSize: 30),
                 historyListItemLayoutBuilder: (controller, value, index) {
                   return InkWell(
-                    onTap: () => controller.select(value.text),
+                    onTap: () async => await controller.select(value.text),
```

Comment:
older versions don't have asyc-await, maybe don't work ?

Reply:
_save() added to select method which needs to be await